src/swarm/addrs.js

```diff
@@ -20,28 +22,30 @@ module.exports = (createCommon, options) => {
 
       common.setup((err, factory) => {
         expect(err).to.not.exist()
-        factory.spawnNode((err, node) => {
+
+        spawnNodesWithId(2, factory, (err, nodes) => {
           expect(err).to.not.exist()
-          ipfs = node
-          done()
+          ipfsA = nodes[0]
+          ipfsB = nodes[1]
+          ipfsA.swarm.connect(ipfsB.peerId.addresses[0], done)
         })
       })
     })
 
     after((done) => common.teardown(done))
 
     it('should get a list of node addresses', (done) => {
-      ipfs.swarm.addrs((err, multiaddrs) => {
+      ipfsA.swarm.addrs((err, multiaddrs) => {
```

Comment:
Should we change `multiaddrs` to `peers` or `peersInfo`?

I was not understanding the `forEach` validation, but after getting into the code, I noticed that in fact `ipfs.swarm.addrs()` returns an array of `PeerInfo`. This way, calling this `multiaddrs` seems confusing. I think we should also update in the [SPEC](https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/SWARM.md#ipfsswarmaddrscallback)

Reply:
I changed to `peerInfos` as there's already precedent for that name in `swarm.peers` docs.